packages/block-library/src/post-comments/index.php

```diff
@@ -26,7 +26,22 @@ function render_block_core_post_comments( $attributes, $content, $block ) {
 	setup_postdata( $post );
 
 	ob_start();
+	// There's a deprecation warning generated by WP Core.
+	// Ideally this deprecation is removed from Core.
+	// In the meantime, this removes it from the output.
+	add_filter(
+		'deprecated_file_trigger_error',
+		function() {
+			return false;
+		}
+	);
 	comments_template();
+	add_filter(
+		'deprecated_file_trigger_error',
+		function() {
+			return true;
+		}
```

Comment:
Shouldn't this be a saved reference to whatever filter was there before?

Reply:
Actually found a better way  using remove_filter

Reply:
Oh, totally. D'oh. :)